src/PythonMigrationViewExtension/Controls/VisualDifferenceViewer.xaml

```diff
@@ -0,0 +1,74 @@
+﻿<Window x:Class="Dynamo.PythonMigration.Controls.VisualDifferenceViewer"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        xmlns:diffplex="clr-namespace:DiffPlex.Wpf.Controls;assembly=DiffPlex.Wpf"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
```

Comment:
let's also set a min `width` and min `height` ?
Don't think a max is needed, so we can skip that one